PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingRequest.kt

```diff
@@ -17,10 +17,10 @@ package com.braintreepayments.api
  */
 data class PayPalMessagingRequest(
     var amount: Double? = null,
-    var placement: PayPalMessagingPlacement? = null,
+    var pageType: PayPalMessagingPageType? = null,
     var offerType: PayPalMessagingOfferType? = null,
```

Comment:
Similar question with these values, if we can set default values for these properties instead of `null` that would make it a bit cleaner.

Reply:
We are defaulting to `null` here as well because that's what the [underlying SDK does](https://github.com/paypal/paypal-messages-android/blob/d3293c07ed758724700d48642f1159e576c748e3/library/src/main/java/com/paypal/messages/config/message/PayPalMessageData.kt#L19-L20). I don't think we want to provide defaults here since the underlying SDK defaults to null.

Reply:
Yeah part of wrapping the SDK though is to allow us to enforce API best practices where it makes sense, and null values aren't the best when a default value is possible. Though if `null` does have semantic meaning I'd agree we should keep. Just making sure we aren't moving on autopilot here.

Reply:
For sure - yeah it seems like `null` does have a semantic meaning for the messaging SDK so we can roll with that as well vs forcing a default on merchants. We wouldn't want to specify a default page type as any default may not actually be accurate for where a merchant may place this notice, so null makes sense there. Same with offer type we wouldn't want to specify that if a merchant doesn't want to use a default as that would determine what offer type their customer is shown.